### DISCUSSION TITLE
Add a menu link to the plugin settings page, accessible on production

### DISCUSSION THREAD

**Discussion by timmyomahony** [timmyomahony/craft-matrix-field-preview#76] [Ideas]
See #75 for context. Usually the "settings" section is hidden on production via `allowAdminChanges=false`. This means the Matrix Field plugin settings are not easily available. I've made it possible to access them directly via `/admin/matrix-field-preview` but this isn't very user friendly. 

Three options:

- Do nothing, just continue to access directly via URL on production
- Add a  link in the menu
- Add a link in the utilities

**Comment by DTesch-Level-R** [comment]
I think the best option would be under the "Utilities"-Tab.

**Reply by craftedsystems** [reply]
Moving the settings to _Utilities_ feels weird and does not solve the update/migrataion issue, when using multiple databases.

**Comment by craftedsystems** [comment]
What about a config file? I would prefer to commit the settings to be able to share and deploy them.

**Reply by devinpitcher** [reply]
I agree. Since fields are stored in the config, I feel these settings should also be stored in the config.

**Comment by craftedsystems** [comment]
Hint: if a custom path to the control panel is set, the settings path changes, too.
Example: `/admin/matrix-field-preview` ⇒ `/cp/matrix-field-preview/settings`

**Comment by timmyomahony** [comment]
FYI, I'm closing this discussion and moving the `project.yaml` discussion over to https://github.com/weareferal/craft-matrix-field-preview/discussions/114
